### PR TITLE
WIP: Media Card component

### PR DESCRIPTION
**Pull request by tlhfckoctbcr**

### CODE REVIEW

File: lib/src/components/media-card.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        .wrapper {
+            display: flex;
+            justify-content: center;
+            flex-wrap: wrap;
+            margin-top: 6.25rem;
+        }
+
+        .container {
+            background: white;
+            width: 19.18rem;
```
Comment (tlhfckoctbcr) [inline lib/src/components/media-card.ts]: Standardize this, either by rounding up to 20rem units, OR by using `aspect-ratio` CSS ruleset.

File: lib/src/components/media-card.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const { LitElement, html, css } = window.Lit;
+
+class MediaCard extends LitElement {
+    static styles = css`
+        
+        .wrapper {
+            display: flex;
+            justify-content: center;
+            flex-wrap: wrap;
+            margin-top: 6.25rem;
```
Comment (tlhfckoctbcr) [inline lib/src/components/media-card.ts]: What is this margin-top value for?

File: lib/src/components/media-card.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            width: 19.18rem;
+            height: 24rem;
+            margin: 1rem;
+            border: 0.25rem solid black;
+            border-radius: 1rem;
+        }
+
+        .container-image {
+            height: 12rem;
+            margin-bottom: 1rem;
```
Comment (tlhfckoctbcr) [inline lib/src/components/media-card.ts]: Please use --space variable.

File: lib/src/components/media-card.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            background-size: cover;
+            border-radius: 0.75rem 0.75rem 0 0;
+        }
+
+        .container-content {
+            display: flex;
```
Comment (tlhfckoctbcr) [inline lib/src/components/media-card.ts:35]: Tell me what `display: flex` is, and how the subsequent rulesets effect your content.

File: lib/src/components/media-card.ts [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            justify-content: space-between;
+            padding: 0rem 0.5rem;
+        }
+
+        ::slotted([slot="title"]) {
+            text-transform: uppercase;
+        }
+
+        ::slotted([slot="description"]) {
+            font-size: 14px;
```
Comment (tlhfckoctbcr) [inline lib/src/components/media-card.ts]: Fix `font-size` to use rem units.

File: lib/src/components/media-card.ts [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+        
+    `;
+
+    render() {
+        return html`
+        
+        <div class="wrapper">
+            <div class="container">
+                <div class="container-image" style="background-image: url('https://picsum.photos/300/200');">
```
Comment (tlhfckoctbcr) [inline lib/src/components/media-card.ts]: What is this?

File: lib/src/components/media-card.ts [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+const { LitElement, html, css } = window.Lit;
+
+class MediaCard extends LitElement {
+    static styles = css`
+        
+        .wrapper {
+            display: flex;
+            justify-content: center;
+            flex-wrap: wrap;
```
Comment (tlhfckoctbcr) [inline lib/src/components/media-card.ts]: Be capable of explaining `flex` and `flex-wrap`.

File: lib/src/components/media-card.ts (hunk 2 above)
Comment (sp92231) [reply lib/src/components/media-card.ts]: It was the top margin for the wrapper, which was not needed so I removed it.

File: lib/src/components/media-card.ts (hunk 1 above)
Comment (sp92231) [reply lib/src/components/media-card.ts]: Standardized to 20rem.

File: lib/src/components/media-card.ts (hunk 5 above)
Comment (sp92231) [reply lib/src/components/media-card.ts]: Changed to 0.825rem.

File: lib/src/components/media-card.ts (hunk 3 above)
Comment (sp92231) [reply lib/src/components/media-card.ts]: Changed to --space variable.

File: lib/src/components/media-card.ts (hunk 6 above)
Comment (sp92231) [reply lib/src/components/media-card.ts]: Changed to a slotted image.

File: lib/src/components/media-card.ts (hunk 4 above)
Comment (sp92231) [reply lib/src/components/media-card.ts:35]: Creates a Flex container and the rulesets control the layout of the container.

File: lib/src/components/media-card.ts (hunk 7 above)
Comment (sp92231) [reply lib/src/components/media-card.ts]: Flex controls the item growing and shrinking and flex-wrap defines the wrapping style and whether or not it goes to the next line.

**Comment by sp92231** [comment]
Updated code and resolved all comments.

**Pull request merged**